manual/README.md

```diff
@@ -136,6 +137,8 @@ The following setup assumes a Unix-like environment.
 
 11. Run the Ethereum sidecar. For example:
     ```shell
+    # This example shows usage of an Ethereum Sepolia RPC for Mezo testnet.
+    # Remember to use an Ethereum Mainnet RPC for Mezo mainnet.
     mezod ethereum-sidecar \
      --ethereum-sidecar.server.ethereum-node-address=wss://eth-sepolia.g.alchemy.com/v2/<redacted>
```

Comment:
I propose to use mainnet configuration for all examples as it is for `docker` deployment.

```suggestion
    # This example shows usage of an Ethereum RPC for Mezo mainnet.
    mezod ethereum-sidecar \
     --ethereum-sidecar.server.ethereum-node-address=wss://eth-mainnet.g.alchemy.com/v2/<redacted>
```